doctr/models/utils.py

```diff
@@ -83,7 +82,7 @@ def conv_sequence(
         list of layers
     """
     # No bias before Batch norm
-    kwargs['use_bias'] = kwargs.get('use_bias', not(bn))
+    kwargs['use_bias'] = kwargs.get('use_bias', not (bn))
```

Comment:
same here

Reply:
ok will do. sorry about that, i let Pycharm correct everything without checking :/